firmware/Core/Inc/telecommands/i2c_telecommand_defs.h

```diff
@@ -0,0 +1,7 @@
+#include <stdint.h>
+#include "telecommands/telecommand_definitions.h"
+#include "main.h"
+
+
+uint8_t TCMDEXEC_scan_i2c_device(const char *args_str, TCMD_TelecommandChannel_enum_t tcmd_channel,
```

Comment:
Can we rename from `scan_i2c_device` to `scan_i2c_bus`? We're not scanning a single device, but rather scanning for any/all devices on a bus.

Reply:
Done.